android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java

```diff
@@ -104,7 +115,26 @@ public void setSrc(FastImageViewWithUrl view, @Nullable ReadableMap source) {
                     .load(imageSource.getSourceForLoad())
                     .apply(FastImageViewConverter.getOptions(source))
                     .listener(new FastImageRequestListener(key))
-                    .into(view);
+                    .into(new SimpleTarget<Drawable>() {
+                        @Override
+                        public void onResourceReady(@NonNull Drawable resource, @android.support.annotation.Nullable Transition<? super Drawable> transition) {
+                            if (resource instanceof BitmapDrawable) {
+                                Bitmap bitmap = ((BitmapDrawable) resource).getBitmap();
+
+                                ThemedReactContext context = (ThemedReactContext) view.getContext();
+                                RCTEventEmitter eventEmitter = context.getJSModule(RCTEventEmitter.class);
+                                WritableMap resourceData = new WritableNativeMap();
+                                resourceData.putInt("width", bitmap.getWidth());
+                                resourceData.putInt("height", bitmap.getHeight());
+                                eventEmitter.receiveEvent(viewId, REACT_ON_LOAD_EVENT, resourceData);
```

Comment:
```suggestion
                                eventEmitter.receiveEvent(viewId, REACT_ON_LOAD_EVENT, resourceData);
                                eventEmitter.receiveEvent(viewId, REACT_ON_LOAD_END_EVENT, new WritableNativeMap());
```

This ensures the onLoadEnd event is fired.